frontends/election-manager/src/screens/ballot_screen.tsx

```diff
@@ -161,6 +154,44 @@ export function BallotScreen(): JSX.Element {
     isAbsentee,
   });
 
+  const onPreviewRendered = useCallback((pageCount) => {
+    setBallotPages(pageCount);
+  }, []);
+
+  const ballotWithCallback = useCallback(
+    (onRendered: (pageCount: number) => void) => {
+      return (
+        <HandMarkedPaperBallot
+          ballotStyleId={ballotStyleId}
+          election={election}
+          electionHash={electionHash}
+          ballotMode={ballotMode}
+          isAbsentee={isAbsentee}
+          precinctId={precinctId}
+          locales={locales}
+          onRendered={onRendered}
+        />
+      );
+    },
+    [
+      ballotMode,
+      ballotStyleId,
+      election,
+      electionHash,
+      isAbsentee,
+      locales,
+      precinctId,
+    ]
+  );
+
+  const printBallot = useCallback(async () => {
+    await printElementWhenReady(ballotWithCallback, {
+      sides: 'two-sided-long-edge',
+      copies: ballotCopies,
+      screenDisplayNone: false,
```

Comment:
With PagedJs, it does not. I could not tell you for certain why. Currently, all our paged ballots are rendered to `printBallotRef` or a child of `printBallotRef`, which does not have `display: none;`. PagedJs simply doesn't work in a `display: none;` container. I wonder if PagedJs is injecting some media queries somewhere although I have looked in their codebase and couldn't find any.

Reply:
Hmm, weird. I would be interested to look at the DOM in the inspector and see what's going on with the Paged.js target element. I think this is worth understanding.

Reply:
OK, I took another look. It's because all of our apps have the following:

```css
/* Foundational */
html,
body,
#root {
  height: 100%;
...
```

And everything being printed is being rendered outside of the `#root` in the `#print-root`, so the ballots are below the visible page. To keep it simple, in this case, I think we shouldn't make any of them `display: none;`, and just remove that styling entirely.

Reply:
In case anything like PagedJs or a future tool relies on layout and absolute positioning (can't imagine why it would), using `visibility` seems safest:
```css
const PrintOnly = styled.div`
  @media screen {
    visibility: hidden;
  }
`;
```